topic-operator/src/test/java/io/strimzi/operator/topic/ZkTopicStoreTest.java

```diff
@@ -14,18 +16,17 @@
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 
-import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
 
 public class ZkTopicStoreTest extends TopicStoreTestBase {
-
     private static Vertx vertx;
-
     private EmbeddedZooKeeper zkServer;
-    private Zk zk;
+    private Zk zkClient;
 
     @Override
     protected boolean canRunTest() {
-        return false; // test was disabled
+        return true;
```

Comment:
Is this `canRunTest` construct still needed?

Reply:
I don't think so, let me get rid of it.